### PR TITLE
Add ZWES0054E to error msg

### LINKED ISSUE #2873: Issue with docs.zowe.org/stable/troubleshoot/app-framework/zss-error-codes/
## Description
error ZWES0054E message not documented 
## Pages to Update
(https://docs.zowe.org/stable/troubleshoot/app-framework/zss-error-codes/)
## Additional context
ZWESASTC ending in error
2023/03/23-04:45:01.15 ZWESASTC 00FC3200(00D3) 00AF81A0 ZWES0050I ZIS AUX Server starting, version is 2.5.0+20221130
2023/03/23-04:45:01.15 ZWESASTC 00FC3200(00D3) 00AF81A0 ZWES0054E ZIS AUX Server started in wrong key 8

message ZWES0054E ZIS AUX Server started in wrong key 8
message ZWES0054E ZIS AUX Server started in wrong key 8

ZWESISTC: i have a message:
IZPP0004E Init step 'AUX' failed - AUX guest not started, RC = 12, RSN = 0x00300008
ZWES0017W Plug-in 'ZSSDPROVIDER ' failure - plug-in init RC = 148

Jorn Thyssen from Rocket suggest the following:
Looks like you’re missing the PPT entry for ZWESAUX:
https://docs.zowe.org/v2.2.x/user-guide/configure-xmem-server/#zowe-auxiliary-service

Below the entry in the sysparm SCHED00, thougt that is OK

SYS1.PARMLIB(SCHED00)
PPT PGMNAME(ZWESIS01) /* ZOWE APF SERVER /
KEY(4) / ERIK 10SEP2019 */
NOSWAP

I added now also the following lines into SCHED00
PPT PGMNAME(ZWESAUX)
KEY(4)
NOSWAP

But I still get the same message ZWES0054E ZIS AUX Server started in wrong key 8.
Is an IPL of the system requiered for this change?

No, It can be loaded dynamically with :/SET SCH=00
You can check afterwards with /D PPT

Furthermore next steps have to be made to fix the problem:

    authPluginLib was missing in the ZWESASTC JCL and also in the ZWESISTS JCL
    The IZPSRGSP,IZPSRGAD,ZWESVUSR need read access to the profile OMVSAPPL

Thereafter the ZWESASTC started task was started successfully by the ZWESISTS started task.
Those steps are not documented.

### PR DESCRIPTION
**Pull request by samanthasusu**
Signed-off-by: samanthasusu <Wen.Ting.Su@ibm.com>
- Add the ZWES0054E which was not documented before.

### Your checklist for this pull request
🚨Please review the [guidelines for contributing](../docs/contribute/contributing.md) to this repository.

- [ ] If the changes in this PR is part of the next future release, make this pull request against the **docs-staging** branch which will be published at the next release boundary. If the changes in this PR are part of the current release, use the default base branch, **master**. For more information about branches, see https://github.com/zowe/docs-site/tree/master#understanding-the-doc-branches. 
      
- [ ] If this PR relates to GitHub issues in `docs-site` or other repositories, please list in Description, prefixed with **close**, **fix** or **resolve** keywords.

### Description (including links to related git issues)
Please describe your pull request.
fix #2873 
:heart:Thank you!

### CODE REVIEW

**Comment by samanthasusu** [comment]
@1000TurquoisePogs hi Sean, would you please review this tip and see if it can be merged into doc? Thank you!

**Comment by samanthasusu** [comment]
I suggest we add this error message to the troubleshooting section as I just find that it is not available on the live site yet. Would you mind taking this PR over to get it merged Andrew @janan07 ? Thanks!

File: docs/troubleshoot/app-framework/zss-error-codes.md [hunk 1]
```diff
@@ -367,3 +367,15 @@ The following error message codes may appear on ZSS log. Use the following messa
...
+
+  ZIS AUX Server started in wrong key 8
+
+  **Reasons:**
+
+  authPluginLib was missing in the `ZWESASTC JCL` and the `ZWESISTS JCL`.
+
+  **Action:**
+ 
+  The `IZPSRGSP`,`IZPSRGAD`, and `ZWESVUSR` need read access to the profile `OMVSAPPL`. Then the ZWESASTC started task was started successfully by the ZWESISTS started task.
```
Comment (1000TurquoisePogs) [inline docs/troubleshoot/app-framework/zss-error-codes.md:380]: IZP is some third party thing, not part of Zowe.
And `ZWESVUSR` is just the default name of Zowe's user account.

This should read as:

> When the Zowe STC userid has read access to the profile SAF `OMVSAPPL`, the ZWESASTC task will be able to successfully be started on an as-needed basis from ZWESISTC.

File: docs/troubleshoot/app-framework/zss-error-codes.md [hunk 2]
```diff
@@ -367,3 +367,15 @@ The following error message codes may appear on ZSS log. Use the following messa
 
 No action required.
 
+### ZWES0054E
+
+  ZIS AUX Server started in wrong key 8
+
+  **Reasons:**
+
+  authPluginLib was missing in the `ZWESASTC JCL` and the `ZWESISTS JCL`.
```
Comment (1000TurquoisePogs) [inline docs/troubleshoot/app-framework/zss-error-codes.md:376]: ZWESISTC, not ZWESISTS

**Review by 1000TurquoisePogs — CHANGES_REQUESTED** [review]

**Review by 1000TurquoisePogs — APPROVED** [review]

**Pull request merged**